### ISSUE TITLE
text-tags parameters not getting passed for embed unclaimed draft

### ISSUE THREAD

**Issue by oliviermills** [hellosign/hellosign-php-sdk#6]
The text tags seem to get ignored in this call...

``` PHP
try {

            $client = new HelloSign\Client(HELLOSIGN_API_KEY);
            $request = new HelloSign\SignatureRequest;
            $request->enableTestMode();
            $request->setRequesterEmail($user_email);
            $request->addSigner($user_email, $user_name);
            $request->addSigner($client_email, $client_fname . " " . $client_lname);
            $request->addFile($contract_pdf_filename);
            $request->setUseTextTags(true);
            $client_id = HELLOSIGN_CLIENT_ID;

            ///// UNCLAIMED DRAFT
            $draft_request = new HelloSign\UnclaimedDraft($request, $client_id);        
            $draft_request->setIsForEmbeddedSigning(true);
            $response = $client->createUnclaimedDraft($draft_request);
            $unclaimDraft = get_object_vars($response);
            $claim_url = $draft_request->getClaimUrl();
```

**Comment by oliviermills** [comment]
the text-tags is not being set for embedded signing either:  (below `setUseTextTags(true)` gets ignored... you have to override in the sdk commenting out `line 57` of the `EmbeddedSignatureRequest` class

``` PHP

    $request->addFile( $filename );

    $request->addSigner('email@gmail.com', "Signer Name");
    $request->addSigner('email@hotmail.com', "Signer2 Name");

    $request->setUseTextTags(true);

    $request->setSubject('My First embedded signature request with a template');
    $request->setMessage('Awesome, right?');
    $client_id = HELLOSIGN_CLIENT_ID;   
    $embedded_request = new HelloSign\EmbeddedSignatureRequest($request, $client_id);
    $embedded_request->enableTestMode();
    $response = $helloSignclient->createEmbeddedSignatureRequest($embedded_request);
```

**Comment by BHSPitMonkey** [comment]
These are now fixed as of [3.2.7](https://github.com/HelloFax/hellosign-php-sdk/releases/tag/3.2.7). Thanks for the report!
